src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -88,10 +88,13 @@ public void EmitSymbolRef(string symbolName, int size, bool isPCRelative, int de
         }
 
         [DllImport(NativeObjectWriterFileName)]
-        private static extern void EmitFrameInfo(IntPtr objWriter, string methodName, int startOffset, int endOffset, int blobSize, byte[] blobData);
-        public void EmitFrameInfo(string methodName, int startOffset, int endOffset, int blobSize, byte[] blobData)
+        private static extern void EmitFrameInfo(IntPtr objWriter, string methodName, int startOffset, int endOffset, int blobSize, byte[] blobData,
+                                                 string personalityFunctionName, int LSDASize, byte[] LSDA);
+        public void EmitFrameInfo(string methodName, int startOffset, int endOffset, int blobSize, byte[] blobData,
+                                  string personalityFunctionName = null, int LSDASize = 0, byte[] LSDA = null)
```

Comment:
JIT actually never sets the flag, but VM (JitInterface, allocUnwindInfo) set it -- https://github.com/dotnet/coreclr/blob/master/src/vm/jitinterface.cpp#L11181
I changed object writer behavior in a consistent to the flag. Now we don't emit such personality routine since such exception flag is not set. Only when such flag is set, the personality function symbol is emitted -- of course, the string should be supplied or assertion fails. https://github.com/dotnet/llilc/blob/master/lib/ObjWriter/objwriter.cpp#L385-L392

So, in order words, when you define EH region (in LSDA) with a personality function in CoreRT, you should set the flag as well accordingly.